scripts/base/init-bare.zeek

```diff
@@ -2268,6 +2268,52 @@ const RPC_status = {
 	[RPC_UNKNOWN_ERROR] = "unknown"
 };
 
+## Generic analyzer confirmation info record.
+##
+## .. zeek:see:: analyzer_confirmation_info
+type AnalyzerConfirmationInfo: record {
+	## The connection related to this confirmation, if any.
```

Comment:
Related to my comment above I would suggest to explicitly say when this is set: only for protocol analyzers, or generally if there a connection semantically associated to this activity?